source/proxy_connection.c

```diff
@@ -1150,6 +1150,7 @@ static int s_setup_proxy_tls_env_variable(
             proxy_verify_peer_string != NULL && aws_string_eq_c_str(proxy_verify_peer_string, "off")) {
```

Comment:
Can we change this to AWS_PROXY_NO_VERIFY_PEER and then just check for definition rather than the value itself.